src/diffusers/models/modeling_utils.py

```diff
@@ -829,6 +808,9 @@ def _get_model_file(
             and version.parse(version.parse(__version__).base_version) >= version.parse("0.17.0")
         ):
             try:
+                if commit_hash is not None and revision is None:
+                    revision = commit_hash
```

Comment:
Note that this will cause the warning logs in this function to print the commit hash instead of the revision

Reply:
Good catch!